File: java/warmup-2/array123.java
/* Given an array of ints, return true if .. 1, 2, 3, .. appears in the array 
 * somewhere.
 */
public boolean array123(int[] nums) {
    for(int i = 0; i < nums.length - 2; i++) {
        if(nums[i] == 1 && nums[i+1] == 2 && nums[i+2] == 3)
            return true;
    }
    
    return false;
}

File: python/warmup-2/array123.py
# Given an array of ints, return True if .. 1, 2, 3, .. appears in the array 
# somewhere.
def array123(nums):
    for i in xrange(len(nums) - 2):
        if nums[i] == 1 and nums[i+1] == 2 and nums[i+2] == 3:
            return True
                  
    return False
